src/dispatch/plugins/dispatch_slack/case/messages.py

```diff
@@ -287,6 +287,14 @@ def create_action_buttons_message(
     return Message(blocks=signal_metadata_blocks).build()["blocks"]
 
 
+def create_signal_metadata_blocks(signal_metadata_blocks: list[Block], message: str) -> list[Block]:
```

Comment:
```suggestion
def create_genai_signal_message_metadata_blocks(signal_metadata_blocks: list[Block], message: str) -> list[Block]:
```

Reply:
I'd make the function name more specific to genai as we include `:magic_wand: *GenAI Alert Analysis` in the section text.